site/snippets/signers/privy.ts

```diff
@@ -32,7 +28,7 @@ await embeddedWallet.switchChain(sepolia.id);
 // Get a viem client from the embedded wallet
 const eip1193provider = await embeddedWallet.getEthereumProvider();
 const privyClient = createWalletClient({
-  account: embeddedWallet.address,
+  account: embeddedWallet.address as `0x${string}`,
```

Comment:
do you need to cast to `Hex` here? is address field just `string` type?

Reply:
Ran linter with ajay + customer tested this and it works so merging